### ISSUE TITLE
Window instantiation

### ISSUE THREAD

**Issue by cdecompilador** [BurntSushi/winapi-util#6]
I'd like to contrib to the project and extend the utility for window instantiation and control, is the project still continued?

**Comment by BurntSushi** [comment]
In general, yes, I'm open to adding things. I have some requirements/requests though:

* Every API needs to be documented.
* I'm not a Windows user and very unfamiliar with it in general. I've learned portions of it that I needed to, but other aspects of Windows I'm less familiar with. So this probably means that the burden for documenting things is higher because ultimately I'm the one who needs to maintain the crate.
* The crate should continue to have no dependencies other than `winapi`.
* One of my goals for this crate was specifically to expose _safe_ wrappers for Windows API routines. I'd like to continue that if possible. If `unsafe` is required, then it should be carefully designed.
* In general, I'd like any APIs exposed to assume as little as possible and permit the full range exposed by the Windows API routines. (Or at least, make it possible to expand the crate later easily to accommodate such things.) There is often a tension between this and reducing `unsafe` usage, so this is really the core challenge.

Otherwise, I used the name "util" in the crate name to indicate this is an approximate dumping ground for things.
